README.md

```diff
@@ -21,39 +20,24 @@ This project contains Ansible code that creates a baseline in an existing kubern
   - kustomize such that data and homes directories are mounted on cas nodes and on compute server instances
   - Deploy [Viya Monitoring for Kubernetes](https://github.com/sassoftware/viya4-monitoring-kubernetes)
   - Deploy MPP or SMP CAS servers
+
 - Manage Viya Deployments
   - Organize and persist config for any number of Viya deployments across namespaces, cluster, or cloud providers.
 
-### Prerequisites
-
-This tool supports running both from ansible installed on your local machine or via a docker container. The Dockerfile for the container can be found [here](Dockerfile)
+## Prerequisites
 
-#### Ansible
+### Operational knowledge of 
 
-- terraform
-- ansible
-- unzip
-- tar
-- kubectl
-- kustomize
-- python
-- pip
-- openshift pip module
-- kubernetes  pip module
-- dnspython pip module
-- community.kubernetes ansible-galaxy module
-- ansible.posix ansible-galaxy module
-- helm
-- git
+- [Ansible](https://docs.ansible.com/ansible/latest/user_guide/index.html#getting-started)
+- [Docker](https://www.docker.com/)
+- [Kubernetes](https://kubernetes.io/docs/concepts/)
+- Cloud Provider
 
-#### Docker
+### Technical
 
-- docker
-- git
+- [Ansible and docker dependencies](docs/user/Dependencies.md)
 
-note: For dependency versions and changing versions see [Dependency Versions](docs/DEPENDENCY-VERSIONS.md)
-
-#### Infrastructure
+### Infrastructure
 
 Prior to running this playbook some infrastructure needs to be in place
 
```

Comment:
Under "Kubernetes cluster: You can either bring your own K8s cluster..." Add GCP project given GCP content has been added to this document. If pushed prior to GCP project release, I'd still recommend adding it here and putting in a caveat about the expected release date.

Reply:
updated